### ISSUE TITLE
Won't work any more since Skype has discontinued this service

### ISSUE THREAD

**Issue by kamranahmedse** [Stichoza/skype-status-php#5]
It seems like Skype has [discontinued this service!](https://support.skype.com/en/faq/FA605/how-do-i-set-up-the-skype-button-to-show-my-status-on-the-web-in-skype-for-windows-desktop) and one cannot find out the status of a user this way.

**Comment by Stichoza** [comment]
That is sad 🙁 I knew this time would come.

So I came across [this js script](http://www.skypeassets.com/i/scom/js/skype-uri.js) in the new skype buttons source. I haven't yet looked at this, but I think there would be something that can make this package alive again :smile:

**Comment by kamranahmedse** [comment]
Yeah, the source of buttons, that is what I myself looked into before opening this issue so that I may create a PR but it seems like there is nothing in there to determine the status of user.

Anyways good luck. I will be looking forward to see what you cone up with :-)
